docs/source/key_components/dashboard_visualization.rst

```diff
@@ -0,0 +1,109 @@
+Dashboard Visualization
+=======================
+
+Env-dashboard is a post-experiment visualization tool, aims to provide
+more intuitive environment information, which will guide the design of
+the algorithm and continually fine-tuning.
+
+Feature List
+------------
+Basically, each scenario has 2 parts of visualization: inter epoch and intra epoch. User could switch between them freely.
```

Comment:
2 kinds of features?  inter-epoch and intra-epoch?

Reply:
inter-epoch-view, intra-epoch-view? It's better to unify them in the summary and in the sub-titles

Reply:
2 kinds of views. How about change it to : 
Feature List

Views: Intra/Inter
Index Selection
Sampling Ratio